src/commonMain/kotlin/exchange.dydx.abacus/state/supervisor/NetworkHelper.kt

```diff
@@ -255,6 +255,10 @@ class NetworkHelper(
                             }
 
                             else -> {
+                                val geoStatus = parsedHeaders?.get("geo-origin-status")
+                                if (geoStatus == "restricted") {
+                                    indexerRestriction = UsageRestriction.http403Restriction
```

Comment:
unfamiliar with how abacus handles all 403s, what exactly changes app-wide? Are we just forcing 403 as a shortcut despite being in the `else` block?